### DISCUSSION TITLE
Binding Directly To Model & Relationship

### DISCUSSION THREAD

**Discussion by ahosker** [livewire/livewire#5703] [1. Help]
Im having real trouble with Model Biding on a relationship.

The [documentation](https://laravel-livewire.com/docs/2.x/properties#binding-models) says it's possible "Additionally, you can bind to models within an Eloquent Collection."

Client Model.
hasOne income
hasOne credithistory

On the livewire form, if a user updates **income**. No problems. If the user updates **credithistory** I get the error:
```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'credithistory' in 'field list'
```

It seems Laravel is acting like credithistory is an array of data instead of a relation.

I'm seeing no difference in how I implement these two relationships but one wont work and the other does.

I'm bringing the models in on mount:
```
    public function mount($client)
    {
        $this->client = Client::with('user', 'income', 'credithistory')->find($client);
    }
```

I'm doing the validation rules the same:
```
            'client.income.income_child_benefit' => 'nullable|numeric',
            'client.credithistory.bankrupcy' => 'nullable|boolean',
```

I'm doing the saving the same

```
    public function save()
    {

        //Validate Input
        $this->validate();

        //Save Child Models
        $this->client->income->save();
        $this->client->credithistory->save();

        if ($this->client->isDirty()) {
            $this->client->save();
        }
    }
```

I'm doing the input the same:
```
<x-input label="income_child_benefit" wire:model="client.income.income_child_benefit" type="number"/>
<x-toggle label="bankrupcy" wire:model.defer="client.credithistory.bankrupcy" />
```

**Comment by joshhanley** [comment]
@ahosker can you share your `Client` model class?
